app/javascript/components/volunteer/Profile.jsx

```diff
@@ -87,13 +85,13 @@ class Profile extends React.Component {
                     </div>
                 </div>
 
-                <h3 className="pt5">Skills</h3>
-                {this.props.user.skills}
+                <h3 className="pt5 f3">Skills</h3>
+                <p className="f5">{this.props.user.skills}</p>
```

Comment:
We should replace this with a list of all the skills, the same way you are doing it for projects, since we are depreciating the `skills` field for users.